CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # TRILLIAN Changelog
 
+## HEAD
```

Comment:
Any preference on what to put here? "HEAD", "master", etc? Looking to communicate that it's what you'll get if you pull master and build Trillian.

Reply:
I think HEAD is unambiguous?